runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/DoFnOperator.java

```diff
@@ -380,7 +386,6 @@ public void close() throws Exception {
       }
     }
     checkFinishBundleTimer.cancel(true);
```

Comment:
@aljoscha I had the doubt if this one should be moved to dispose too, given that close can eventually not be called. WDYT ?

Reply:
yes, you're right. This can also be moved to `dispose()`. 👍